### DISCUSSION TITLE
Page hierarchy

### DISCUSSION THREAD

**Discussion by sebrame** [webstudio-is/webstudio-community#176] [Q&A]
Is there a way to reorder the pages in the Pages Panel hierarchy?

**Comment by kof** [comment]
Currently no, but if you are seeing too many pages and want to better organize, you can use folders and folders don't need to have a slug. They can be purely organizational, without affecting the URL.

**Comment by kof** [comment]
Subscribe this issue https://github.com/webstudio-is/webstudio/issues/403
